src/coreComponents/codingUtilities/StringUtilities.cpp

```diff
@@ -29,7 +29,7 @@ namespace stringutilities
 /**
  * String tokenizing function
  **/
-string_array Tokenize( const std::string & str, const std::string & delimiters )
+string_array Tokenize( const string & str, const string & delimiters )
```

Comment:
You may want to change also "const string" to "string const" as the rule said:

![image](https://user-images.githubusercontent.com/45685596/106248289-1159dc80-6211-11eb-90e8-84ec6a501756.png)

Reply:
Position of `const` is not restrained to `string`. It will probably be more efficient to deal with `const` position all at once, for all the argument types.

Reply:
Yes....we should do the const enforcement in a stand-alone PR.